.gitignore

```diff
@@ -4,3 +4,4 @@ npm-debug.log
 .eslintcache
 dist/
 output/
+examples/ExamplePdf.md
```

Comment:
Could you refrain from touching .gitignore please?

Reply:
Noted I will remove it